vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -329,6 +345,11 @@ class SpyreModelRunner(BaseSpyreModelRunner[SamplingInputBatch,
 
     def __init__(self, vllm_config: VllmConfig, is_driver_worker: bool,
                  rank: int):
+        # Normally we would check if the modelconfig supports multimodal inputs
+        # by comparing the model config against the registry here
+
+        # We also calculate the multimodal budget; this is probably important
+        # see scratch/projects/vllm/vllm/v1/worker/utils.py (in the budget)
```

Comment:
is this a file path on your workspace?

Could link out to a specific tag of vllm instead

Reply:
This is actually a stale comment and was mostly relevant to static batching I think - I'll remove it!